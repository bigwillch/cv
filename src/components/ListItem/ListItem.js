import React from 'react';

export const ListItem = ({
  children,
}) => (
  <li>
    <span>{children}</span>
  </li>
);
