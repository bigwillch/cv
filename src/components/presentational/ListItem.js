import React from 'react'

let ListItem = (props) => {

  return (
    <li>
      <span>{props.children}</span>
    </li>
  )
}

export default ListItem
