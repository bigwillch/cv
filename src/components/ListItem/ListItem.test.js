import React from 'react';
import { ListItem } from './ListItem';
import toJson from 'enzyme-to-json';

describe('ListItem', () => {
  const wrapper = mount(
    <ListItem>
      <p>List Item Content</p>
    </ListItem>
  );

  it('should wrap children in a span', () => {
    expect(wrapper.first().find('li').childAt(0).type()).toEqual('span');
  });

  it('should match snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});