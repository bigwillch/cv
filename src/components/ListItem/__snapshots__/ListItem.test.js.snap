// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should match snapshot 1`] = `
<ListItem>
  <li>
    <span>
      <p>
        List Item Content
      </p>
    </span>
  </li>
</ListItem>
`;
